fetch_index.py

```diff
@@ -27,7 +27,8 @@
 ITEM = "item"
 SITES = "sites"
 LISTS = "lists"
-ITEMS = "items"
+ITEMS = "list_items"
+DRIVES = "drive_items"
```

Comment:
`ITEMS` is a bit confusing. Maybe considering rename to `LIST_ITEMS` and `DRIVE_ITEMS`?

Reply:
Correct. It should be `LIST_ITEMS`. Updating this